katcp/core.py

```diff
@@ -1,35 +1,32 @@
 # katcp.py
 # -*- coding: utf8 -*-
 # vim:fileencoding=utf8 ai ts=4 sts=4 et sw=4
-# Copyright 2009 National Research Foundation (South African Radio Astronomy Observatory)
+# Copyright 2010 National Research Foundation (South African Radio Astronomy Observatory)
 # BSD license - see LICENSE for details
 
 """Utilities for dealing with KAT device control language messages."""
 
-from __future__ import division, print_function, absolute_import
-
+from __future__ import absolute_import, division, print_function
+import future
```

Comment:
`future` is a 3rd party module, perhaps place it under 3rd party imports. Do not confuse `from future import standard_library` and `future`...

Read here: https://python-future.org/reference.html?highlight=standard_library#id13

Reply:
I prefer the current version, simply because it puts all the imports for Python 2 support in one place. One Day ™️ we can just rip out the entire code block in one go when we go Python 3 only.

Reply:
I suppose it would be much easier to rip them off when that One Day ™️. 
Thanks for the input.